tests/test_yaml_validation.py

```diff
@@ -0,0 +1,43 @@
+import os
+import unittest
+
+
+from yaml2sbml.yaml_validation import validate_yaml
+
+from jsonschema.exceptions import ValidationError
+
+class TestYamlValidation(unittest.TestCase):
+
+    def setUp(self):
+        this_dir, _ = os.path.split(__file__)
+        self.test_folder = os.path.join(this_dir, 'test_yaml2sbml')
+
+    def test_validate_yaml_valid_1(self):
+        file_in = os.path.join(self.test_folder, 'ode_input1.yaml')
+        validate_yaml(file_in)
+
+    #Should throw an error because of extra fields in conditions
+    def test_validate_yaml_valid_2(self):
```

Comment:
Shouldn't be a `with self.assertRaises(...):` be here, too?

Reply:
Yes, but it doesn't fail :/

Reply:
To be honest, I do not see, why the input yaml should result in an error... @martamatos can you help me here? :)